src/Module/Magic.php

```diff
@@ -71,41 +71,76 @@ protected function rawContent(array $request = [])
 
 		$addr = $_REQUEST['addr'] ?? '';
 		$dest = $_REQUEST['dest'] ?? '';
+		$bdest = $_REQUEST['bdest'] ?? '';
 		$owa  = (!empty($_REQUEST['owa'])  ? intval($_REQUEST['owa'])  : 0);
```

Comment:
Since you're at it, please use the provided `$request` array instead of the `$_REQUEST` superglobal.
```suggestion
		$addr  = $request['addr'] ?? '';
		$dest  = $request['dest'] ?? '';
		$bdest = $request['bdest'] ?? '';
		$owa   = intval($request['owa'] ?? 0);
```